languages/clojure/docs/implementing-a-concept-exercise.md

```diff
@@ -1,5 +1,57 @@
-# How to implement an Clojure concept exercise
+# How to implement a Clojure concept exercise
 
-TODO: describe how to implement a concept exercise for the Clojure track. For inspiration, check out the [C# version of this file][csharp-implementing].
+This document describes how to implement a concept exercise for the Clojure track. As this document is generic, the following placeholders are used:
 
-[csharp-implementing]: ../../csharp/docs/implementing-a-concept-exercise.md
+- `<SLUG>`: the name of the exercise in kebab-case (e.g. `anonymous-methods`).
+- `<NAME>`: the name of the exercise in PascalCase (e.g. `AnonymousMethods`).
+
+Before implementing the exercise, please make sure you have a good understanding of what the exercise should be teaching (and what not). This information can be found in the exercise's GitHub issue. Having done this, please read the [Clojure concept exercises introduction]//TODO.
+
+To implement a concept exercise, the following files must be created:
+
+<pre>
+languages
+└── clojure
+    └── exercises
+        └── concept
+            └── &lt;SLUG&gt;
+                ├── .docs
+                |   ├── instructions.md
+                |   ├── introduction.md
+                |   ├── hints.md
+                |   └── after.md (optional)
+                ├── .meta
+                |   ├── config.json
+                |   └── Example.clj
+                ├── &lt;NAME&gt;.clj
+                └── &lt;NAME&gt;_test.clj
+</pre>
+
+## Step 1: adding track-specific files
+
+These files are specific to the Clojure track:
+
+- `<NAME>.clj`: the Clojure file.
+- `<NAME>_test.clj`: the test suite.
+
+## Step 2: adding common files
+
+How to create the files common to all tracks is described in the [how to implement a concept exercise document][how-to-implement-a-concept-exercise].
```

Comment:
The [how-to-implement-a-concept-exercise] link is missing (see bottom of https://raw.githubusercontent.com/exercism/v3/master/languages/csharp/docs/implementing-a-concept-exercise.md).

Reply:
@EduardoBautista corrected, thank u~